app.rb

```diff
@@ -14,25 +14,6 @@ def initialize
     @people = []
   end
 
```

Comment:
- [ ] Also, it would be better if we move this code `student = Student.new(age, name, parent_permission)` and `teacher = Teacher.new(age, name, specialization)` into the create_person method

Reply:
Done.